README.md

```diff
@@ -27,6 +27,7 @@ Additional documentation for this sample can be found on [Microsoft Docs](https:
 
 ## Prerequisites
 
+- Xcode 13 or greater
```

Comment:
Should it also include min iOS version?

Reply:
As long as you have Xcode 13, you have the minimum iOS version installed.